src/storage-client/src/controller.rs

```diff
@@ -320,6 +320,16 @@ pub trait StorageController: Debug + Send {
         exports: Vec<(GlobalId, ExportDescription<Self::Timestamp>)>,
     ) -> Result<(), StorageError>;
 
+    /// For each identified export, update its `StorageSinkConnection`.
+    ///
+    /// n.b. this method is not more generalized because `StorageSinkConnection`
+    /// encapsulates much of what you could reasonably change about a sink's
```

Comment:
This n.b section confused me more than it helped clarify things. What would it mean to be more generalized? I would omit this altogether unless I'm missing something

Reply:
Sorry if this was unclear––by more general, I had meant modifying `StorageSinkDesc` instead of `StorageSinkConnection`. Doesn't pain me to remove this, though.